devops/provision/Dockerfile

```diff
@@ -39,7 +39,15 @@ RUN set -ex \
  && git clone https://github.com/WildMeOrg/wbia-plugin-flukematch.git \
  && git clone https://github.com/WildMeOrg/wbia-plugin-finfindr.git \
  && git clone https://github.com/WildMeOrg/wbia-plugin-deepsense.git \
- && git clone https://github.com/WildMeOrg/wbia-plugin-pie-v2.git
+ && git clone https://github.com/WildMeOrg/wbia-plugin-pie-v2.git 
+
+RUN set -ex \
```

Comment:
I had originally git cloned in the style of the plugins above rather than the third-party plugins below but it was not getting latest changes. maybe due to lack of a develop branch? Anyway, git cloning in this way un-broke the build.